docs/backup-restore.md

```diff
@@ -24,6 +24,34 @@ That means that you need to create a consistent snapshot by dumping all tables i
 For example, you can use `mysqldump` command with `--single-transaction` option in MySQL and `pg_dump` command in PostgreSQL to achieve that.
 Or when you use Amazon RDS (Relational Database Service) or Azure Database for MySQL/PostgreSQL, you can restore to any point within the backup retention period with the automated backup feature, which satisfies the requirement.
 
+## Cosmos DB
+
+You can create Cosmos DB account using `Continuous` backup policy to create continuous backups.
+
+The easiest way to take a transactionally-consistent backup for Scalar DB on Cosmos DB is to implement [scalar-admin](https://github.com/scalar-labs/scalar-admin) interface properly in your application, you can easily pause the application without losing ongoing transactions.
+You can use the middle value of the pause time as a restore point.
+
+You can restore the backup based on the [azure official guide](https://docs.microsoft.com/en-us/azure/cosmos-db/restore-account-continuous-backup#restore-account-portal) and you must change the default consistency to `STRONG` after restoring the data.
+
+## DynamoDB
+
+ScalarDB schema loader enables Point-in-time recovery (PITR) for each table in a DynamoDB while creating the schema. It will help you to create continuous backups.
+
+The easiest way to take a transactionally-consistent backup for Scalar DB on DynamoDB is to implement the [scalar-admin](https://github.com/scalar-labs/scalar-admin) interface properly in your application, you can easily pause the application without losing ongoing transactions.
+You can use the middle value of the pause time as a restore point.
+
+You can restore the tables one by one using the following steps from the Amazon DynamoDB console,
+
+    A. Restore the PITR (Point-in-time recovery) backup of tables except `scalardb.metadata` on the basis of [AWS official guide](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/PointInTimeRecovery.Tutorial.html#restoretabletopointintime_console).
+        
+    B. Create the backup of previously restored tables (A) using the [AWS official guide](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Backup.Tutorial.html#backup_console).
+        
+    C. Delete all tables except `scalardb.metadata` table and previously restored tables (B).
+        
+    D. Restore the previously created backup (B) using the actual table name (previously deleted tables (C)) on the basis of [AWS official guide](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Restore.Tutorial.html#restoretable_console).
+
+You must enable continuous backup and auto-scaling using the scalardb schema loader or Amazon DynamoDB console. The schema tool doesn't remake the existing tables.
```

Comment:
Several comments.
* It's too late to say `you must` here. If it is a must, we should say in the first sentence.
* why auto-scaling is also a must? (I don't really know)
* schema loader or schema tool. Not consistent
* the part `You must ... using the` sounds a bit weird to me as an English. Maybe `using` -> `with`?

Reply:
>  It's too late to say you must here. If it is a must, we should say in the first sentence.

I think it's the right place. After the restore, continuous backup and auto-scaling will not be enabled automatically, so we need to execute the schema tool for that, otherwise, we need to enable it manually.

> why auto-scaling is also a must? (I don't really know)

I think yes, otherwise we will face performance issues.

> schema loader or schema tool. Not consistent

Sorry for the inconvenience.
Document updated with `schema loader `

> the part You must ... using the sounds a bit weird to me as an English. Maybe using -> with?

Sorry for the inconvenience.
Document updated with `using -> with` 01ae20a